osu.Game/Online/Multiplayer/IMultiplayerClient.cs

```diff
@@ -42,6 +42,14 @@ public interface IMultiplayerClient
         /// <param name="user">The user.</param>
         Task UserKicked(MultiplayerRoomUser user);
 
+        /// <summary>
+        /// Signals that a user has been invited into a multiplayer room.
+        /// </summary>
+        /// <param name="invitedBy">Id of user that invited the player.</param>
+        /// <param name="roomID">Id of the room the user got invited to.</param>
+        /// <param name="password">Password to join the room.</param>
+        Task Invited(int invitedBy, long roomID, string password);
```

Comment:
This is actually how it worked on stable, and was an intentional consideration because the invites were links sent out via chat. If the host wanted to stop an invite link working, they could simply change the password.

I'm not sure if we need to change this in the short term or not if it works. Now that invites are sent to a specific user, I guess it does make more sense for that invite to continue working even if the room password is changed, so I wouldn't be against changing this if it's simple enough.

Reply:
As you say, if they're not links anymore, password-changing affecting existing invites makes little logical sense to me. I'm not heavily invested though. More trying to figure out what this feature should even be since it is criminally underspecced and this PR thread shows it (the discussion is being dragged multiple places).

Reply:
Would it be ok to change this to use a server signed token instead of a password?

Reply:
Since password is already working, I'd personally be fine with sticking with it until it's seen as an issue.